### PR TITLE
Update the avatar of IDP icons in linked social accounts and delete options

### PR DESCRIPTION
**Pull request by ShanChathusanda93**
### Purpose
* Updated the IDP avatar in the linked social accounts section in myaccount. 
* Updated the logic for IDP delete options.

### CODE REVIEW

**Review by brionmario — APPROVED** [review]

File: modules/theme/src/themes/default/views/card.overrides [hunk 1]
```diff
@@ -182,6 +182,11 @@
                     }
                     .first-column {
                         padding-left: @settingsCardPadding;
+                        .associations-list-avatar-wrapper {
+                            display: flex;
+                            alignItems: center;
+                            columnGap: 5px;
+                        }
```
Comment (yasinmiran) [inline modules/theme/src/themes/default/views/card.overrides:189]:
```suggestion

                        .associations-list-avatar-wrapper {
                            display: flex;
                            align-items: center;
                            column-gap: 5px;
                        }
```

File: modules/theme/src/themes/default/views/card.overrides [hunk 2]
```diff
@@ -182,6 +182,11 @@
                     }
                     .first-column {
                         padding-left: @settingsCardPadding;
+                        .associations-list-avatar-wrapper {
```
Comment (yasinmiran) [inline modules/theme/src/themes/default/views/card.overrides:185]: This should be in the `list.overrides` file right?

**Review by yasinmiran — CHANGES_REQUESTED** [review]

File: apps/myaccount/src/components/federated-associations/federated-associations.tsx [hunk 3]
```diff
@@ -232,22 +233,24 @@ export const FederatedAssociations: FunctionComponent<FederatedAssociationsProps
...
-                                                <List.Header>
-                                                    {
-                                                        federatedAssociation.idp.displayName
-                                                        || federatedAssociation.idp.name
-                                                    }
-                                                </List.Header>
-                                                <List.Description>
-                                                    { linkedAttribute }
-                                                </List.Description>
+                                                <div style={{display: "flex", alignItems: "center", columnGap:"5px"}}>
```
[the remark this answers is not in the document]
Comment (ShanChathusanda93) [reply apps/myaccount/src/components/federated-associations/federated-associations.tsx]: Moved to `list.overrides` file

File: modules/theme/src/themes/default/views/card.overrides (hunk 1 above)
Comment (ShanChathusanda93) [reply modules/theme/src/themes/default/views/card.overrides]: Applied the changes.

**Review by yasinmiran — APPROVED** [review]

**Pull request merged**